CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 7.7.0 (XXX XX, 2023). Tested on Artifactory 7.55.10
+
+IMPROVEMENTS:
+
+* resource/artifactory_scoped_token: adds reference token exposure on scoped token creation
```

Comment:
* resource/artifactory_scoped_token: adds `reference_token` attribute for the resource.